projects/ADuCM360_demo_cn0338/src/Cmd.cpp

```diff
@@ -197,7 +197,9 @@ void Cmd_ReadData(void)
 {
    while (uart_rx_head != uart_rx_tail) {
       switch (uart_rx_queue[uart_rx_head]) {
-      case ' ' ... '~':
+      case ' ' ... '[':
+      case '\\':
+      case ']' ... '~':
```

Comment:
this looks hack-ish;
i think a better way to fix this would be to move this into it's own code block with an `if ` statement
something like:
```
char c = uart_rx_queue[uart_rx_head];
if (c >= ' ' && c <= '[')
    handle_ascii_data();
```
or something like that